src/declaration.c

```diff
@@ -375,7 +375,7 @@ void AliasDeclaration::semantic(Scope *sc)
     if (s && ((s->getType() && type->equals(s->getType())) || s->isEnumMember()))
         goto L2;                        // it's a symbolic alias
 
-    type = type->addStorageClass(storage_class);
+    type = type->addSTC(storage_class);
```

Comment:
As a quick check, adding `assert(deco)` in `Type::addStorageClass` will break druntime build.